support/catalogs/images.go

```diff
@@ -110,31 +110,45 @@ func imageExistsFn(ctx context.Context, hcp *hyperv1.HostedControlPlane, pullSec
 		if err == nil {
 			return true, nil
 		}
-		if strings.Contains(err.Error(), "manifest unknown") {
+		if strings.Contains(err.Error(), "manifest unknown") || strings.Contains(err.Error(), "access to the requested resource is not authorized") {
 			return false, nil
 		}
 		return false, fmt.Errorf("failed to get image digest: %w", err)
 	}
 }
 
 func computeCatalogImages(releaseVersion func() (*semver.Version, error), imageExists func(string) (bool, error), registryOverrides map[string][]string) (map[string]string, error) {
+	var registries []string
 	version, err := releaseVersion()
 	if err != nil {
 		return nil, err
 	}
 
-	defaultRegistry := "registry.redhat.io/redhat"
-	registries := []string{
-		defaultRegistry,
-	}
+	defaultRegistryURL := "registry.redhat.io"
+	defaultRegistryNamespace := "redhat"
```

Comment:
can we move those to a constant at the beginning of the file for visibility?

Reply:
I'll do in a follow up PR. This blocks the disconnected testing on QE

Reply:
sure!